src/app/tests/suites/certification/Test_TC_OO_1_1.yaml

```diff
@@ -38,38 +38,75 @@ tests:
       command: "readAttribute"
       attribute: "ClusterRevision"
       response:
-          value: 5
+          value: 6
           constraints:
               type: int16u
 
-    - label: "Step 3a: TH reads from the DUT the FeatureMap attribute."
+    - label:
+          "Step 3a: Given OO.S.F00(LT) ensure featuremap has the correct bit set"
       command: "readAttribute"
       attribute: "FeatureMap"
-      PICS: ( !OO.S.F00 && !OO.S.F01 )
+      PICS: ( OO.S.F00 && !OO.S.F02 )
       response:
-          value: 0
           constraints:
               type: bitmap32
-
+              hasMasksSet: [0x1]
     - label:
           "Step 3b: Given OO.S.F00(LT) ensure featuremap has the correct bit set"
```

Comment:
Is this label right? Seems like this should be different

Reply:
Correct Rene! The TP requires: "SHALL be 1 if and only if". The previous implementation was checking that if the feature is enabled, then the bit is 1, but wasn't checking the opposite, for example, the case where the feature is disabled on the device, but still, the feature map sets the bit as 1 (which is not allowed).

Reply:
We should properly figure out what the plan is regarding the 1.1 tests. I know there have been talks about trying to remove these, and replace them by a single unified test instead. 

Also in case we really want to keep these and want to actually make them verify these cases, I'm wondering if it would not be worth simply rewriting these in Python, which I would assume would make them quite more readable.

Reply:
I agree, python would be the best case in this scenario!